acp/TERMINOLOGY.md

```diff
@@ -0,0 +1,9 @@
+## ACP Terminologies
+- 'SourceHub Address' is a Cosmos Address with a specific SourceHub prefix.
+- 'Identity' is a combination of SourceHub Address and a Key-Pair Signature.
+- 'DPI' means 'DefraDB Policy Interface'.
```

Comment:
nitpick: You are not defining what a `DefraDB Policy Interface` is in the terminology section, there is a link to another doc on it in the footer, but a one-liner here might be handy too.

Note: This is currently worsened because the DPI doesn't really say what a DPI is either, it only notes some restrictions and a handful of examples.

Reply:
- [ ] Fix Terms and DPI

Reply:
- [x] More Detailed ACP Doc https://github.com/shahzadlone/defradb/blob/lone/acp-sourcehub-module/acp/ACP.md